enigma/src/main/java/cuchaz/enigma/analysis/index/IndexReferenceVisitor.java

```diff
@@ -139,13 +139,17 @@ public void visitInvokeDynamicInsn(String name, String descriptor, Handle bootst
 		}
 
 		private ReferenceTargetType getReferenceTargetType(int stackDepth) {
+			if (stack == null) { // Unreachable instruction if ASM is to be trusted
```

Comment:
```suggestion
			if (stack == null) { // Unreachable instruction
```

not sure why ASM wouldn't be trusted

Reply:
Thing is I don't quite believe that xmlpull (a public maven artifact) actually contains unreachable instructions.

Reply:
See #575 for a fix for xmlpull. However your fix will also be good to have for cases of genuine unreachable code. There's no ASM bug here anyway.